src/Pages/MembershipApplication/register-page.css

```diff
@@ -1,93 +1,139 @@
-.membership-form {
-  width: 40%;
+.membership {
+  width: 100%;
+}
+
+.form-card {
+ position: relative;
+ left: 50%;
+ top: 50%;
+ transform: translate(-50%,-50%);
+
+ width: 35%;
+ min-width: 400px;
+ 
+ box-shadow: 0 2px 8px rgba(0, 0, 0, 0.26);
+ border-radius: 20px;
+ padding: 1.5rem;
+ background: white;
+ flex: none;
+ overflow: hidden;
+}
+
+.membership-info {
+ width: 40%;
+ float: left;
+ min-width: 625px;
+ font-size: 500%;
+ color: white;
+ background-color: rgba(173, 169, 169, 0.3);
+ text-align: left;
+ top: 50%;
+ left: 50%;
+ transform: translate(0, 50%);
+ height: 50%;
+ vertical-align: middle;
+ padding: 2rem;
+ padding-bottom: 0rem;
 }
 
 #membership-plan-btn {
-  align-items: center;
-  justify-content: center;
+ align-items: center;
+ justify-content: center;
 }
 
 .transition-button-wrapper {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin-top: 30px;
+ display: flex;
+ justify-content: space-evenly;
+ align-items: center;
+ margin-top: 30px;
 }
 
 .unavailable {
-  color: red;
-  font-weight: 500;
+ color: red;
+ font-weight: 500;
 }
 
 #name-field-row{
-  justify-content: space-between;
-  margin-left: 0px;
-  margin-right: 0px;
+ width: 100%;
+ display: inline-block;
+ vertical-align: middle;
+ margin: auto;
+ margin-top: 10px;
 }
 
 .membership-card {
-  max-width: 35%;
-  text-align: center;
+ max-width: 35%;
+ text-align: center;
 }
 
 .membership-plan-row {
-  justify-content: space-evenly;
+ justify-content: space-evenly;
 }
 
 .active-plan {
-  background-color: rgb(188, 255, 186);
+ background-color: rgb(188, 255, 186);
 }
 
 .email-input {
-  width: 100%;
+ width: 100%;
 }
 
 #email-input-container {
-  width: 100%;
-  display: inline-block;
-  vertical-align: middle;
+ width: 100%;
+ display: inline-block;
+ vertical-align: middle;
 }
 
 .membership {
-  display: inline-block;
+ display: inline-block;
 }
 
 .membership-card:hover {
-  background-color: lightblue;
-  cursor: pointer;
+ background-color: lightblue;
+ cursor: pointer;
 }
 
 .img{
-  height: 200px;
-  width: 100%;
+ height: 200px;
+ width: 100%;
 }
 
 .fade-enter {
-  opacity: 0;
+ opacity: 0;
 }
 
 .fade-enter.fade-enter-active {
-  opacity: 1;
-  transition: opacity 200ms ease-in;
+ opacity: 1;
+ transition: opacity 200ms ease-in;
 }
 
 .fade-leave {
-  opacity: 1;
+ opacity: 1;
 }
 
 .fade-leave.fade-leave-active {
-  opacity: 0;
-  transition: opacity 200ms ease-in;
+ opacity: 0;
+ transition: opacity 200ms ease-in;
+}
+
+.recaptcha {
+ margin: 20px 20px 0px 0px;
 }
 
 #change-and-select-btns {
-  background-color: #e1f5fe;
-  border-color: black;
-  color: black;
+ background-color: #e1f5fe;
+ border-color: black;
+ color: black;
 }
 
 #submit-btn {
-  background-color: #8fdcff;
-  border-color: black;
-  color: black;
+ background-color: #8fdcff;
+ border-color: black;
+ color: black;
 }
+
+#login {
+ width: 100%;
+ display: inline-block;
+ text-align: center;
+}
```

Comment:
that means add an empty line at the end of the file . [Why?](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file)
![image](https://user-images.githubusercontent.com/18486562/80543935-c7338a00-8964-11ea-815b-d9fabeb27abb.png)